qiskit_ibm_runtime/accounts/account.py

```diff
@@ -81,6 +83,36 @@ def from_saved_format(cls, data: dict) -> "Account":
             verify=data.get("verify", True),
         )
 
+    def resolve_cloud_instance_crn(self) -> None:
```

Comment:
Can we consider renaming this method? I keep reading it as "Cloud Instance" and keep wondering what it is. May be something like `resolve_crn` or `resolve_service_instance_crn`?